.gitignore

```diff
@@ -23,6 +23,7 @@ npm-debug.log
 /apps/newrelic_agent.log
 /apps/node_modules/
 /apps/npm-debug.log
+/apps/src/code-studio/pd/application/facilitator1819/sharedConstants.js
```

Comment:
What is facilitator1819?

Reply:
Feels a little bit strange to me to be ignoring js changes that are this deep in our tree. I wonder if we'd be better off sticking all of these sorts of files somewhere likes `/apps/src/generated`. Not sure how strongly I feel about this though.

Reply:
It's the [facilitator application for the 2018-2019 school year](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/application/facilitator1819_application.rb). Since we have multiple types of applications now, and we expect the fields to change each year we created a STI table for `pd_applications` and categorize them by type (facilitator, teacher) and year.

Good point. I like it that suggestion. I'll move it higher up in the tree.